crates/cli/src/benchmark.rs

```diff
@@ -45,16 +45,23 @@ pub struct BenchmarkCommand {
     )]
     iterations_per_process: usize,
 
-    /// The format of the output data. Either 'json' or 'csv'.
+    /// Output raw data, rather than the summarized, human-readable analysis
+    /// results.
+    #[structopt(long)]
+    raw: bool,
```

Comment:
Because then everything that takes an `OutputFormat` would have to support this: `summarize` and `effect-size`. The human readable display isn't *just* for benchmark results or *just* for the summaries or *just* for the effect size; it uses *all* of those things to create its output. Adding this human readable output to those other commands would make them duplicate this same output, which would be confusing: "why is this differently named command giving me identical data?" Effectively, the `benchmark` subcommand is the "main" subcommand, often the only one that most people need to ever run, and its `--raw` flag and the other subcommands are relegated to advanced users who want to work with the raw data. (Along these lines, I also considered moving the existing `benchmark` subcommand to `raw-benchmark`, but this just reintroduces all that duplication we had with `in-process-benchmark`.)

Reply:
Ok, yeah, makes sense.